README.md

```diff
@@ -1,2 +1,63 @@
-# KITABOOK
- KITABOOK | Helping new readers hook up with books
+<h1 align="center">
+<br />
+<a href="https://vispaui-docs-development.netlify.app/"><img src="https://raw.githubusercontent.com/Husain01/Kitabook/Dev/assets/KITABOOK.png" alt="Neu UI" width="300"></a>
+
+<br/>
+</h1>
+
+
+# **📚 KITABOOK**
+### KITABOOK | Helping new readers hook up with books
+<br/>
+
+<!-- PROJECT SHIELDS -->
+<p align="center">
+  <!-- <a href="https://github.com/Husain01/Neu-UI/blob/main/LICENSE" target="blank">
```

Comment:
you remove the unwanted code from here

Reply:
Will be adding it later that's why have commented.